modules/microshift-lvms-csi-snapshot-optional.adoc

```diff
@@ -0,0 +1,14 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="microshift-lvms-csi-snapshot-optional_{context}"]
+= LVMS CSI provider and CSI snapshot deployments
+
+{microshift-short} can now be configured to disable the built-in logical volume manager storage (LVMS) Container Storage Interface (CSI) provider or the CSI snapshot capabilities. These capabilities can reduce the use of runtime resources such as RAM, CPU, and storage.
+
+[IMPORTANT]
+====
+Automated uninstallation is not supported as this can cause orphaning provisioned volumes. Without the LVMS CSI driver, the cluster has no knowledge of the underlying storage interface and so it cannot perform provisioning/deprovisioning or mount/unmount operations.
```

Comment:
```suggestion
Automated uninstallation is not supported as this can cause provisioned volumes to become orphaned. Without the LVMS CSI driver, the cluster has no knowledge of the underlying storage interface and so it cannot perform provisioning/deprovisioning or mount/unmount operations.
```

Reply:
include both suggestion